UNL-CSE-AMBASSADORS/VITA
# Appointment signup fix

Added a script and modified the JS to store appointments. Sends an email notification upon completion and notifies the client upon success.

To test:
1. Go to /signup/
2. Sign up
3. Look at confirmation message
4. Make sure the database has data in it
